EIPS/eip-4762.md

```diff
@@ -268,6 +254,13 @@ class AccessSubtree(Container):
     elements: BitVector[256]
 ```
 
+## Block-level operations
+
+* Precompile accounts, system contract accounts and slots of a system contract that are accessed during a system call are considered warm.
+* When (and only when) calling a system contract _via a system call_, the code chunks and account should not appear in the witness.
+* The coinbase account is warmed before the block, and the whole account is accessed
```

Comment:
whole account = basic data + code hash?

Reply:
yes

Reply:
this is because the coinbase might not exist, so to avoid a lot of issues, we decide to touch the whole account so that it's created if it doesn't exist already.

Reply:
what if there is no tx that puts any value to it, and lets say there is no tips that get added to a non existent coinbase, then it should be 158 cleaned up in the end of block?